templates/hash/functions.go

```diff
@@ -48,6 +48,10 @@ func (fns goSharedFuncs) pointerAccessor(field pgs.Field) string {
 	return fmt.Sprintf("&m.%s", fns.Name(field))
 }
 
+func (fns goSharedFuncs) fieldName(field pgs.Field) string {
+	return fmt.Sprintf("%s", fns.Name(field))
```

Comment:
just curious, what is the advantage of doing this over `string(fns.Name(field))`?

Reply:
No reason in particular. I was just following the format of the other functions. I probably could have just used `fns.Name(field).ToString()`